analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ExecutingSqlQueriesTest.cs

```diff
@@ -111,6 +111,13 @@ public class ExecutingSqlQueriesTest
             Enumerable.Empty<MetadataReference>()
                       .Concat(NuGetMetadataReference.MicrosoftEntityFrameworkCore(entityFrameworkVersion))
                       .Concat(NuGetMetadataReference.MicrosoftEntityFrameworkCoreRelational(entityFrameworkVersion));
+
 #endif
+
+        [TestMethod]
+        public void ExecutingSqlQueries_CS_EF6() =>
+            builderCS.AddPaths("ExecutingSqlQueries.EF6.cs")
+                .AddReferences(NuGetMetadataReference.EntityFramework(Constants.NuGetLatestVersion))
```

Comment:
The test name says `EF6`, the NuGet version says `Latest`. We should aligne one or the other. I'm fine with `Latest` so the test name and test case file should be `EF` or `EntityFramework` in that case.

Reply:
The entity framework naming is confusing in many aspects. EF6 is the canonical name of "the latest EF version of the sunsetted EF for the full framework". That one has nothing in common with what was called EF Core. There will be no successor to EF6. EF7 is the successor to EF Core 6 which is another Nuget package. `Constants.NuGetLatestVersion` will never go beyond version 6 for this package. I don't know how to express this best in the test.

Reply:
I didn't want to know 😆 

So the "Latest" is not needed, as there will be no more updates. And `EntityFramework()` should just do the trick ,because it's declared as `EntityFramework(string packageVersion = "6.2.0")`.

Or you can hardcode the current latest `EntityFramework("6.4.4")` in the UT, to match the EF6 suffix.